apps/studio/components/interfaces/ProjectCreation/ProjectCreation.schema.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod'
+
+export const FormSchema = z.object({
+  organization: z.string({
+    required_error: 'Please select an organization',
+  }),
+  projectName: z
+    .string()
+    .trim()
+    .min(1, 'Please enter a project name.') // Required field check
+    .min(3, 'Project name must be at least 3 characters long.') // Minimum length check
```

Comment:
The odd thing is this isn't enforced on project rename, so you can't create a project with a two-character name, but you can rename it to that afterwards

Reply:
ahh the project settings hasnt been converted to use the new RHF stuffs - i'll just add a basic validation for now